incubator/virtualcluster/cmd/kubectl-vc/use.go

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"sigs.k8s.io/multi-tenancy/incubator/virtualcluster/pkg/syncer/conversion"
+)
+
+const vcKubeconfigHome = ".kube/vc/"
+
+var useCommand = cli.Command{
+	Name:      "use",
+	Usage:     "Switch kubeconfig among virtualclusters",
+	ArgsUsage: "VC_NAME",
+	Flags: []cli.Flag{
+		cli.StringFlag{
```

Comment:
If I understand correctly, this name is the vc cr ns name.   How about let user specify the fullname of the vc cr so that we can remove one input parameter and avoid mistakes. 

e.g., Kubectl vc use tenant1-admin/vc-sample1

Reply:
I like that but it does sort of change peoples dev exp around kubeconfigs.